packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -188,23 +187,33 @@ class DevToolsAppState extends State<DevToolsApp> {
   }
 }
 
-class DevToolsScreen {
-  const DevToolsScreen({
-    @required this.screen,
-    this.controllerProvider,
+class DevToolsScreen<T> {
```

Comment:
Document what T is as part of the class level comment. Consider changing it to `C` as it is the Controller which might be a little unexpected.